### DISCUSSION TITLE
Close own pull-request when comments are restricted

### DISCUSSION THREAD

**Discussion by dicearr** [community/community#33575] [Pull Requests]
I opened a pull request a while ago in a repository I'm not a maintainer of.
At some point the maintainer disabled the comments in my pull-request, allowing only comments from contributors.
I'm not 100% sure, but I'd say that since then I don't see the button to close my own pull-request, as the whole form at the bottom of the page has been replaced with a warning explaining the comments limitation.

Is it correct that restricting the comments is causing me not being able to close the pull-request?
Is there a way for me to close it? Or should I ask the maintainer to do it?

**Comment by radar-blip** [comment]
I'm not 100% sure either but it sounds like you're going to have to ask the maintainer to close it.

**Comment by LangLangBart** [comment]
> Is there a way for me to close it? Or should I ask the maintainer to do it?

Can't you just delete your forked repo and the PR will be closed automatically?

```zsh
# Info about open PR's using GitHub command line tool (gh)
gh api graphql --raw-field query='query($filter: String!) {search(query: $filter, type: ISSUE, first: 100) {nodes { ... on PullRequest { author {login} baseRepository{nameWithOwner} headRepository {nameWithOwner} number state title}}}}' --raw-field filter="author:dicearr type:pr state:open" --template '{{- range .data.search.nodes -}}{{- tablerow "STATE" "BASE" "FORK" "PR NUMBER" "AUTHOR" "TITLE" -}}{{- tablerow .state .baseRepository.nameWithOwner .headRepository.nameWithOwner .number .author.login .title -}}{{- end -}}'

# STATE  BASE           FORK           PR NUMBER  AUTHOR   TITLE
# OPEN   mysqljs/mysql  dicearr/mysql  2218       dicearr  Add idleConnectionTimeout to pool options


# delete repo
gh repo delete dicearr/mysql
```

**Reply by dicearr** [reply]
I don't think removing the fork is going to help, this is what I got from the [docs][]:

> If a pull request's author is unresponsive to requests or has deleted their fork, the pull request can still be merged. However, if you want to make changes to a pull request and the author is not responding, you'll need to perform some additional steps to update the pull request.

[docs]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally#modifying-an-inactive-pull-request-locally
